main.go

```diff
@@ -50,7 +50,7 @@ func setupRouter(sqlDb *sqlx.DB) *gin.Engine {
 
 	router.POST("/api/send-career-mail", jobsRepo.SendCareerMail)
 
-	router.GET("/sitemap", sitemapRepo.GenerateSitemap)
+	router.GET("/api/sitemap", sitemapRepo.GenerateSitemap)
```

Comment:
We should update API URL in test also

Reply:
I think we are not testing path in tests. We are testing that path we configure for any endpoint works fine. In short, we are not writing tests for main.go, thus it does not matter much if we update path in tests.

Reply:
And tests not failing confirms that.

Reply:
Yes, we are not testing the path, but it's for similarity.